src/semeio/fmudesign/design_distributions.py

```diff
@@ -220,8 +239,10 @@ def draw_values_lognormal(dist_parameters, numreals, rng, normalscoresamples=Non
             scale=exp(mean),
         )
     else:
-        distribution = scipy.stats.lognorm(s=sigma, scale=exp(mean))
-        values = distribution.rvs(size=numreals, random_state=rng)
+        uniform_samples = generate_stratified_samples(numreals, rng)
+        values = scipy.stats.lognorm.ppf(
+            uniform_samples, s=sigma, loc=0, scale=exp(mean)
```

Comment:
nitpick: instead of using `exp` from math and everything else from scipy, use `np.exp` here while we're at it?

Reply:
I like the nitpick